server/service/src/requisition/request_requisition/insert_program.rs

```diff
@@ -204,12 +204,17 @@ fn generate(
         Some(ProgramIndicatorFilter::new().program_id(EqualFilter::equal_to(&program.id))),
     )?;
 
+    let customer_name_id = StoreRowRepository::new(connection)
+        .find_one_by_id(&ctx.user_id.clone())?
+        .ok_or(RepositoryError::NotFound)?
+        .name_link_id;
```

Comment:
```suggestion
    let customer_name_id = StoreRepository::new(connection)
        .query_by_filter(StoreFilter::new().id(EqualFilter::equal_to(&ctx.store_id)))?
        .pop()
        .ok_or(RepositoryError::NotFound)?
        .name_row
        .id;
```

to get the correct name.id instead of using name link id